Simplify localStorage data stored

elv-zenia opened the pull request:
This pull request addresses the following:
- Only save ABR profile in localStorage data if the user chooses a "custom" playback encryption. Otherwise, save the option and allow elv-abr-profile to generate the profile.
- For restricted DRM, generate the ABR profile using elv-abr-profile and further change to have only Fairplay and Widevine playout formats.
- Remove ABR profile JSON data saved on the client-side.
- Change Active Jobs to Jobs; next phase will use profile service to store the jobs.
- Catch localStorage quota exceeded error and display error message.
- Save total file size from drag/drop data; remove extra file meta call.
- Update elv-client-js to ~~4.0.9~~ 4.0.14.
- Add error log modal with more detailed information.

Relates to #24 
Relates to #21

elv-zenia commented:
@eponymous301 @elv-serban This addresses some of the reported ingest app issues. Working on the next part.

eponymous301 commented on src/stores/IngestStore.js:
```diff
@@ -422,6 +445,25 @@ class IngestStore {
+      if(missingStreams.length > 0) {
+        const response = yield this.ShowWarningDialog({
+          title: "Streams Not Found",
+          description: `No ${missingStreams.join(", ")} streams found in the media file. Would you like to continue the ingest?`
```
Change to "No suitable ... stream(s) found..."

(The server-side heuristic for choosing streams can find an audio stream but decide not use it because e.g. it had an unsupported audio channel count.)

eponymous301 commented on src/components/LeftNavigation.js:
```diff
@@ -5,7 +5,7 @@ const LeftNavigation = () => {
     <nav className="navigation">
       <NavLink to={"/new"} className="navigation__link">Create</NavLink>
-      <NavLink to={"/jobs"} className="navigation__link">Active Jobs</NavLink>
+      <NavLink to={"/jobs"} className="navigation__link">Jobs</NavLink>
```
Maybe add a section specifically for Running Jobs? (running and/or finished but un-finalized? How does app behave if user navigates away while job is running, i.e. to start another ingest? Is it possible that a job would be left unfinalized?)

elv-zenia replied:
So I think a separation of active/inactive jobs will be addressed in the next iteration where active and inactive jobs will be stored differently. Trying to get this to a finished stopping point to come back to the next wave of enhancements.

If the user navigates away while a job is running, the calls from the ingest steps still get made. It's taken care in a higher level component.

eponymous301 replied:
Ok no worries

elv-zenia commented:
Deployed to testv4.

elv-zenia commented:
@eponymous301 I added a fix for clearing the LRO status interval when there's an error, added `active` and `lastUpdatedTime` properties to the job object, and fixed Clear Inactive Jobs so it clears everything but jobs that have been last updated within the last two minutes. Still working on the other items you found.
Just deployed to demo.

elv-zenia commented:
Relates to https://github.com/qluvio/elv-apps-projects/issues/195
Commit a2cbf702732ac2f2621cc1f2d264d32fb8116458
